oak_abi/build.rs

```diff
@@ -26,6 +26,7 @@ fn main() {
             "../oak/proto/log.proto",
             "../oak/proto/oak_abi.proto",
             "../oak/proto/roughtime_service.proto",
+            "../oak/proto/permissions.proto",
```

Comment:
Do we really expect this file to be used as part of the ABI? I was thinking it would only be used as part of the oak_loader binary?

Reply:
It is needed in the Runtime. I have moved it to oak_runtime.